addons/mass_mailing_sms/static/tests/tours/mailing_activities_split.js

```diff
@@ -0,0 +1,39 @@
+/** @odoo-module */
+
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add('mailing_activities_split', {
+    test: true,
+    url: '/web',
+    steps: [
+        {
+            content: 'Open Activity Systray',
+            trigger: '.o-mail-ActivityMenu-counter',
+        }, {
+            content: 'Open Email Activities',
+            trigger: '.o-mail-ActivityGroup :contains("Email Marketing")',
+        }, {
+            content: 'Open Email Marketing record in the kanban view',
+            trigger: '.o_kanban_record:contains("New Email!")',
+            run: () => {
+                if ($(".o_kanban_record:contains(New SMS!)").length != 0) {
+                    console.error('SMS Marketing record should not appear in this view');
+                }
+            },
+        }, {
+            content: 'Open Activity Systray',
+            trigger: '.o-mail-ActivityMenu-counter',
+        }, {
+            content: 'Open SMS Activities',
+            trigger: '.o-mail-ActivityGroup :contains("SMS Marketing")',
+        }, {
+            content: 'Open SMS Marketing record in the kanban view',
+            trigger: '.o_kanban_record:contains("New SMS!")',
+            run: () => {
+                if ($(".o_kanban_record:contains(New Email!)").length != 0) {
+                    console.error('Email Marketing record should not appear in this view');
```

Comment:
Not a super big deal here, but in general never use `==` (loose equality) in JS and always prefer `===` (strict equality), as the former can lead to unexpected results e.g. `[1, 2] == '1,2'`